.github/workflows/ci.yml

```diff
@@ -0,0 +1,89 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!master'
+
+env:
+  BUILDER_VERSION: fixes
```

Comment:
Note that this will change to the builder release tag once my changes there are merged.